src/mds/Server.cc

```diff
@@ -950,6 +950,10 @@ void Server::find_idle_sessions()
 		 << dendl;
 	break;
       }
+
+      if (not mds->locker->is_revoking_any_caps_from((session)->get_client()))
+	continue;
```

Comment:
this is not right. we should do the check for open sessions. as I said, is_revoking_any_caps_from should always return false for stale sessions